examples/dahlia/dot-product.expect

```diff
@@ -1,5 +1,5 @@
 {
-  "cycles": 91,
+  "cycles": 131,
```

Comment:
What caused this change? It looks like the Dahlia backend stopped generating `"static"` annotations for a bunch of groups it did before.

Reply:
For [Dahlia 368](https://github.com/cucapra/dahlia/pull/368), in the static timing map, I removed the associations `mult -> 3 cycles` and `sqrt -> 17 cycles`. 

Perhaps I should've discussed this more beforehand, but my thought process below:

For `sqrt`, the number of iterations is now dependent on the WIDTH provided.
For `mult`, I wasn't able to find anything that says the multipliers for fixed point run in exactly 3 cycles.

Reply:
The multiplier implementation in the standard library takes three cycles. This is important for targeting FPGAs since multipliers are mapped on DSP blocks which must take at least three cycles to perform the multiplication.

If that makes sense, I'd recommend adding back the `mult` stuff. The rationale for `sqrt` is reasonable so no need to change it.

Reply:
Done in [Dahlia 370](https://github.com/cucapra/dahlia/pull/370).